src/main/java/by/itechart/api/controller/UserControllerInfo.java

```diff
@@ -3,29 +3,34 @@
 import by.itechart.api.dto.CreateUserDTO;
 import by.itechart.api.dto.UpdateUserDTO;
 import by.itechart.api.dto.UserDTO;
-import io.swagger.annotations.ApiOperation;
-import io.swagger.annotations.ApiParam;
-import io.swagger.annotations.ApiResponse;
-import io.swagger.annotations.ApiResponses;
+import io.swagger.annotations.*;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
 import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
 
 import java.util.List;
 
 public interface UserControllerInfo {
-    @ApiOperation(value = "Get all users", notes = "This method will return all users from DB")
+    @Operation(security = @SecurityRequirement(name = "basicAuth"))
+    @ApiOperation(value = "Get all users", notes = "This method will return all users from DB", authorizations = {
```

Comment:
It would better to not expose internal implementation details. The one who will use this REST API doesn't need to know where these users come from
```suggestion
    @ApiOperation(value = "Get all users", notes = "This method will return all users", authorizations = {
```